src/features/dailyForecasts/DailyForecastsPage.tsx

```diff
@@ -56,7 +60,8 @@ export const DailyForecastsPage = () => {
     return <div>You are not authenticated!</div>
   }
 
-  const isBtnDisabled = wxDataLoading || selectedStations.length === 0
+  const isBtnDisabled =
+    wxDataLoading || selectedStations.length === 0 || hourlyDataLoading
 
```

Comment:
We need another variable and stick it in the Button component.
How about const isBtnLoading = wxDataLoading || hourlyDataLoading?
also we need to change the button label to tell users we are getting both forecast and actual weather data